### PR TITLE
hide upload details and show correct progress

### LINKED ISSUE #483: Move upload progress out of user's sight
### Expected behaviour
- upload progress is handled somewhere hidden
- the user can make it visible if desired
- multiple file operation with progress shall be displayed
- progress is not only for upload - also delete and so on ....

see how box is doing it - when opened:

screenshot from 2019-01-10 15-47-39

### LINKED ISSUE #484: Upload Multiple Files
### Expected behaviour
- when opening the upload dialog a user shall be capable to select multiple files
- multiple progresses are visible in the upload status - see #483 
- Question: enter file before upload is finished?

### PR DESCRIPTION
**Pull request by scramb**
## Description

I adjusted the upload progress in the files app and hid it successfully in the topbar. A click on the number of files inProgress opens a menu with all current FileActions. Furthermore I was able to cover the MultipleFile-Upload requirement.  
The vuex-store contains all actions currently in progress, the current total progress is derived from it. 

## Related Issue

- Fixes #483 
- Fixes #484 

## Screenshots (if appropriate):
screenshot_2019-01-17 owncloud phoenix 1
screenshot_2019-01-17 owncloud phoenix 2

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [x] Technical debt

## Checklist:

- [x] Code changes

### CODE REVIEW

File: apps/files/src/store.js [hunk 1]
```diff
@@ -69,6 +92,13 @@ const mutations = {
 }
 
 const actions = {
+  updateFileProgress ({ commit }, progress) {
+    if (progress.progress === 100) commit('REMOVE_FILE_FROM_PROGRESS', { name: progress.fileName })
+    commit('UPDATE_FILE_PROGRESS', progress)
```
Comment (tempelgogo) [inline apps/files/src/store.js]: else

File: apps/files/src/components/FilesApp.vue [hunk 2]
```diff
@@ -123,7 +143,6 @@ export default {
     FileActionsTab,
     FileList,
     FileUpload,
-    FileUploadProgress,
```
Comment (DeepDiver1975) [inline apps/files/src/components/FilesApp.vue]: Kill the component as well - please - THX

Comment (DeepDiver1975) [reply apps/files/src/components/FilesApp.vue]: I'll take care ....

Comment (scramb) [reply apps/files/src/components/FilesApp.vue]: allready cleaned that mess

**Pull request merged**